index.html

```diff
@@ -129,7 +129,61 @@ <h2 class="ui divider horizontal">Ils l'utilisent déjà</h2>
         'contract'
       ],
       searchFullText: true,
-      source: data
+      source: data,
+      templates: {
+        message: function(type, message) {
+          html = '<p class="searchError">Aucune API ne correspond à votre recherche</p>'
```

Comment:
@MattiSG : I don't understand, you are talking about data ? It is called once the page loaded.